plugins/chat/app/controllers/chat/api/reads_controller.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+class Chat::Api::ReadsController < Chat::ApiController
+  def update
+    params.require(:channel_id)
```

Comment:
no biggy, but isn't the single line params.require([:channel_id, :message_id]) equivalent?

Reply:
Ah I didn't even realise you could give it an array :laughing: I always tried `params.require(:x, :y)` and got annoyed when it wouldn't work. Will fix